File: PYTHON/Program/Kendaraan.py
class Kendaraan:
    __platnomor = ""
    __merk = ""
    __tahun = 2024
    __warna= ""
    __cckendaraan = 110
    __tangki = 5
    
    def __init__(self, platnomor, merk, tahun, warna, cckendaraan, tangki):
        self.platnomor = platnomor
        self.merk = merk
        self.tahun = tahun
        self.warna = warna
        self.cckendaraan = cckendaraan
        self.tangki = tangki
        
    def get_platnomor(self):
        return self.platnomor

    def set_platnomor(self, platnomor):
        self.platnomor = platnomor

    def get_merk(self):
        return self.merk

    def set_merk(self, merk):
        self.merk = merk

    def get_tahun(self):
        return self.tahun

    def set_tahun(self, tahun):
        self.tahun = tahun

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna
    
    def get_tangki(self):
        return self.tangki

    def set_tangki(self, tangki):
        self.tangki = tangki
    
    def identifikasi(self):
        return f"Merk       :{self.merk}\n- Plat       :{self.platnomor}\n- Tahun      :{self.tahun}\n- Warna      :{self.warna}\n- CC         :{self.cckendaraan}cc\n- Kapasitas  :{self.tangki} liter"

File: CPP/Program/Kendaraan.cpp
#include <iostream>
#include <string>

#pragma once

class Kendaraan {
private:
    std::string platnomor;
    std::string merk;
    int tahun;
    std::string warna;
    int cckendaraan;
    int tangki;

public:
    Kendaraan(std::string platnomor, std::string merk, int tahun, std::string warna, int cckendaraan, int tangki) 
        : platnomor(platnomor), merk(merk), tahun(tahun), warna(warna), cckendaraan(cckendaraan), tangki(tangki) {}

    std::string get_platnomor() const {
        return platnomor;
    }

    void set_platnomor(const std::string& platnomor) {
        this->platnomor = platnomor;
    }

    std::string get_merk() const {
        return merk;
    }

    void set_merk(const std::string& merk) {
        this->merk = merk;
    }

    int get_tahun() const {
        return tahun;
    }

    void set_tahun(int tahun) {
        this->tahun = tahun;
    }

    std::string get_warna() const {
        return warna;
    }

    void set_warna(const std::string& warna) {
        this->warna = warna;
    }

    int get_tangki() const {
        return tangki;
    }

    void set_tangki(int tangki) {
        this->tangki = tangki;
    }

    virtual void identifikasi() const {
        std::cout << "- Merk       : " << merk << "\n"
                  << "- Plat       : " << platnomor << "\n"
                  << "- Tahun      : " << tahun << "\n"
                  << "- Warna      : " << warna << "\n"
                  << "- CC         : " << cckendaraan << "cc\n"
                  << "- Kapasitas  : " << tangki << " liter" << std::endl;
    }
};